build/cli.js

```diff
@@ -139,10 +140,13 @@ async function buildDocuments(
       appendTotalFlaws(builtDocument.flaws);
     }
 
-    fs.writeFileSync(
-      path.join(outPath, "index.html"),
-      renderHTML(document.url, { doc: builtDocument })
-    );
+    if (noHTML) {
```

Comment:
```suggestion
    if (!noHTML) {
```

no?

Reply:
Stupid me that was called `noJSON` initially ...

Reply:
No idea how that disappeared 😱 But that happened when I renamed the parameter. This PR is even more broken.